ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/OpenSSL102/tasks/main.yml~

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
`main.yml~`?   Seems odd

Reply:
It's a backup file that shouldn't have been in my add - removing ...